.github/workflows/build.yml

```
@@ -112,10 +112,13 @@ jobs:
     - name: Prepare Windows MinGW
       if: contains(matrix.os, 'windows') && contains(matrix.name, 'mingw')
       run: |
+        C:\msys64\usr\bin\pacman -Sy --noconfirm
+        C:\msys64\usr\bin\pacman -S --needed --noconfirm mingw-w64-x86_64-toolchain mingw-w64-x86_64-cppwinrt
```

```suggestion
        pacman -Sy --noconfirm
        pacman -S --needed --noconfirm mingw-w64-x86_64-toolchain mingw-w64-x86_64-cppwinrt
```
Is the full path needed here?

---

Yes, msys2 is installed on the runner image, but not added to PATH https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md#msys2